src/main/java/com/puppycrawl/tools/checkstyle/xpath/AttributeNode.java

```
@@ -182,6 +184,27 @@ public DetailAST getUnderlyingNode() {
         throw throwUnsupportedOperationException();
     }
 
+    /**
+     * Creates nodes for children. Attribute node has no children, so
+     * this method throws {@code UnsupportedOperationException}.
+     *
+     * @return never
```

probably better to say "throw unsupported exception"

---

done

---

not done.

---

Sorry for mistype. Fixed.